src/com/sap/piper/cm/ChangeManagement.groovy

```diff
@@ -9,15 +9,19 @@ public class ChangeManagement implements Serializable {
 
     private script
 
+    private cmHome
+
     protected GitUtils gitUtils
 
-    public ChangeManagement(def script, def gitUtils) {
+    public ChangeManagement(def script, def cmHome, def gitUtils) {
```

Comment:
What is the reason for introducing cmHome? We can rely on a proper setup of the docker image, alternativly (not that good) on a proper setup of the build machine - as long as the docker images are not in place.

If we assume an environment variable ```CMCLIENT_HOME``` and ```ln -s $CMCLIENT_HOME/bin/cmclient /usr/local/bin/cmclient``` there is no need for another way to configure it.

Reply:
This PR is not finished. As I told you the step that you did implement is not tested, and there is tests missing for this step, therefore the coverage check fails. Would you like to do it, or should I do it?

Reply:
I will delete cmHome, so far I did test without docker.

Reply:
>Would you like to do it, or should I do it?
I will take care.